content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -74,6 +75,8 @@ An _output entry_ (`outs`) consists of these fields:
 - `nfiles`: If a directory, number of files inside.
 - `cache`: Whether or not this file or directory is <abbr>cached</abbr> (`true`
   by default, if not present). See the `--no-commit` option of `dvc add`.
+- `desc`: User description for this output. This doesn't affect any DVC
+  operations.
```

Comment:
Also, is it correct that for .dvc files `desc` is a subfield in `outs`? Seems odd since it's a top-level field in dvc.yaml files.

Reply:
It is both.

Reply:
Sorry, didn't get that. I guess you're saying that yes, this is correct: `desc` is at the `outs` level in .dvc files, but at the top-level for dvc.yaml. This "inconsistency" seems unintuitive to me.

Reply:
desc can be for stage or for specific output

Reply:
@jorgeorpinel Let's not focus on this too much, this is minor for docs.

Reply:
`desc` may be minor but 1. that doesn't mean its OK to have an error in docs if we see it (not the case here but I wasn't sure until now) and 2. this is making me realize the way we present outputs vs dvc.yaml in https://dvc.org/doc/user-guide/dvc-files-and-directories is confusing because outputs are only expanded on in the .dvc file section... which is kind of a major. I'll address that elsewhere.